cmd/index.go

```diff
@@ -142,3 +170,83 @@ func pathExist(path string) bool {
 	_, err := os.Stat(path)
 	return err == nil || !os.IsNotExist(err)
 }
+
+func keyName(table *types.Table, fields []string) string {
+	segments := append([]string{strings.ToUpper(table.Name.String())}, fields...)
+	segments = append(segments, "ID")
+	return strings.Join(segments, "_")
+}
+
+func indexDML(table *types.Table, level, max int) [][]IndexDMLPair {
+	allLevelPairs := indexLevel(table, level)
+	for level, pairs := range allLevelPairs {
+		allLevelPairs[level] = randMax(pairs, max)
+	}
+	return allLevelPairs
+}
+
+func indexLevel(table *types.Table, level int) [][]IndexDMLPair {
+	allLevelPairs := make([][]IndexDMLPair, 0, level)
+	allLevelFieldLists := make([][][]string, 0, level)
+	pairs := make([]IndexDMLPair, 0, len(table.Columns))
+	fieldLists := make([][]string, 0, len(table.Columns))
+	for _, column := range table.Columns {
+		fields := []string{column.Name.String()}
+		fieldLists = append(fieldLists, fields)
+		if column.Key == "" {
+			pairs = append(pairs, fields2Pair(table, fields))
+		}
+	}
+	allLevelPairs = append(allLevelPairs, pairs)
+	allLevelFieldLists = append(allLevelFieldLists, fieldLists)
+
+	for i := 1; i < level; i++ {
```

Comment:
i <= level?

Reply:
It does be `i < level` as the index of slice starts from 0.